src/main/java/team/themoment/hellogsmv3/global/security/data/HeaderConstant.java

```diff
@@ -0,0 +1,12 @@
+package team.themoment.hellogsmv3.global.security.data;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum HeaderConstant {
+    X_H_ENV("X-HG-ENV");
```

Comment:
enum type 에 오타가 있는 것 같은데요, 확인 부탁드립니다.
```suggestion
    X_HG_ENV("X-HG-ENV");
```

Reply:
확인 감사드립니다 suggestion 반영하겠습니다.